app.js

```diff
@@ -0,0 +1,42 @@
+const inputAuthor = document.querySelector('.input-author');
+const inputTitle = document.querySelector('.input-title');
+const addBtn = document.querySelector('.add-btn');
+const inputData = document.querySelector('.input-data');
+
+const displayBooks = () => {
+  inputData.innerHTML = '';
+  localStorage.getItem('book');
+  const bookItem = JSON.parse(localStorage.getItem('book') || '[]');
+  bookItem.forEach(({ title, author, id }) => {
+    const div = document.createElement('div');
+    div.innerHTML = `<div>title: ${title}</div>
+                    <div>author: ${author}</div><br>
+                    <button class='remove-btn' id='${id}'>Remove</button>
+                    <hr>`;
+    inputData.appendChild(div);
+  });
+};
+
+const deleteBook = (el) => {
+  const books = JSON.parse(localStorage.getItem('book'));
+  localStorage.setItem('book', JSON.stringify(books));
+  const { id } = el;
+  localStorage.setItem('book', JSON.stringify(books.filter((book) => book.id !== id)));
+  displayBooks();
+};
+
+addBtn.addEventListener('click', () => {
+  const books = JSON.parse(localStorage.getItem('book') || '[]');
+  const id = Math.random().toString(36).substr(0, 5);
+  books.push({ title: inputTitle.value, author: inputAuthor.value, id });
+  localStorage.setItem('book', JSON.stringify(books));
+  displayBooks();
+  inputAuthor.value = '';
+  inputTitle.value = '';
+});
```

Comment:
- [ ] Everything is working perfectly, however let us ensure that we are not adding empty data in our array, kindly validate the input fields
![cpppp1](https://user-images.githubusercontent.com/55635977/133193325-3a468748-be26-44e4-8339-094f0009985d.png)

Reply:
Thank you so much for the review. I am working on it ASAP.